maintainer_scripts/create_readme.sh

```diff
@@ -0,0 +1,534 @@
+The license information given below is obtianed from opam.
+The Coq Platform team does no double check this information.
+
+**Note on multiple licenses:** 
+In case several licenses are given below, it is not clearly specified what this means.
+It could mean that parts of the software use one license while other parts use another license.
+It could also mean that you can choose between the given licenses.
+Please clarify the details with the homepage of the package.
+
+**Note:** The package list is also available as [CSV](${RESULT_FILE_CSV#${DOC_PATH}/}).
+
+**Note:** Click on the triangle to show additional information for a package!
+
+<br>
+
+## **Coq Platform ${COQ_PLATFORM_VERSION} with Coq ${COQ_PLATFORM_COQ_TAG} "base level"**
+
+The **base level** is mostly intended as a basis for custom installations using
+opam and contains the following package(s):
+
+EOT
+
+for package in ${PACKAGES_BASE}
+do
+  html_package_opam "${package}" base
+done
+
+cat >> ${RESULT_FILE_MD} <<EOT
+<br>
+
+## **Coq Platform ${COQ_PLATFORM_VERSION} with Coq ${COQ_PLATFORM_COQ_TAG} "IDE level"**
+
+The **IDE level** adds an interactive development environment to the **base level**.
+
+For beginners, e.g. following introductory tutorials, this level is usually sufficient.
+If you install the **IDE level**, you can later add additional packages individually
+via \`opam install <package-name>\` or rerun the Coq Platform installation script
+and choose the full or extended level.
+
+The **IDE level** contains the following package(s):
+
+EOT
+
+for package in ${PACKAGES_IDE}
+do
+  html_package_opam "${package}" ide
+done
+
+cat >> ${RESULT_FILE_MD} <<EOT
+<br>
+
+## **Coq Platform ${COQ_PLATFORM_VERSION} with Coq ${COQ_PLATFORM_COQ_TAG} "full level"**
+
+The **full level** adds many commonly used coq libraries, plug-ins and
+developments.
+
+The packages in the **full level** are mature, well maintained
+and suitable as basis for your own developments.
+See the Coq Platform [charter](charter.md) for deatils.
+
+The **full level** contains the following packages:
+
+EOT
+
+for package in ${PACKAGES_FULL}
+do
+  html_package_opam "${package}" full
+done
+
+cat >> ${RESULT_FILE_MD} <<EOT
+<br>
+
+## **Coq Platform ${COQ_PLATFORM_VERSION} with Coq ${COQ_PLATFORM_COQ_TAG} "optional packages"**
+
+The **optional** packages have the same maturity and maintenance level as the
+packages in the full level, but either have a **non open source license** or
+depend on packages with non open source license.
+
+The interactive installation script and the Windows installer explicitly ask
+if you want to install these packages.
+
+The macOS and snap installation bundles always include these packages.
+
+The following packages are **optional**:
+
+EOT
+
+for package in ${PACKAGES_OPTIONAL}
+do
+  html_package_opam "${package}" optional
+done
+
+cat >> ${RESULT_FILE_MD} <<EOT
+<br>
+
+## **Coq Platform ${COQ_PLATFORM_VERSION} with Coq ${COQ_PLATFORM_COQ_TAG} "extended level"**
```

Comment:
This section should probably specify if and where the extended level packages are included in the installers (similarly to what is done for the optional packages).